perses/app/setup_relative_calculation.py

```diff
@@ -493,6 +502,18 @@ def run_setup(setup_options):
 
     _logger.info(f"Getting setup options from {yaml_filename}")
     setup_options = getSetupOptions(yaml_filename)
+
+    #### configure platform
+    platform = utils.get_fastest_platform()
+    platform_name = platform.getName()
+    platform = openmm.Platform.getPlatformByName(platform_name) 
+    if platform_name in ['CUDA','CPU']:
+        platform_precision = 'mixed'
+        platform.setPropertyDefaultValue('Precision', platform_precision)
+    else:
+        _logger.warning(f'Platform {platform_name} may not support mixed precision, therefore using default')
```

Comment:
this code is redundant with https://github.com/choderalab/perses/blob/4ee5f1cceb3e733bd42e743cadd248e944177ff0/perses/dispersed/utils.py#L59